docs/source/release_notes.rst

```diff
@@ -5,6 +5,7 @@ Release Notes
 Future Release
 ==============
     * Enhancements
+        * Add ``IsWorkingHours`` and ``IsLunchTime`` transform primitives (:pr:`2130`)  
```

Comment:
This got missed earlier, but lines 16 and 17 thanking the contributors below should be indented with a single indent so they align with the first `*` in front of `* Testing Changes`.